components/browser/menu/src/main/java/mozilla/components/browser/menu/WebExtensionBrowserMenuBuilder.kt

```diff
@@ -12,14 +12,14 @@ import mozilla.components.browser.menu.item.BrowserMenuDivider
 import mozilla.components.browser.menu.item.BrowserMenuImageText
 import mozilla.components.browser.menu.item.NO_ID
 import mozilla.components.browser.menu.item.ParentBrowserMenuItem
+import mozilla.components.browser.menu.item.WebExtensionBrowserMenuItem
 import mozilla.components.browser.menu.item.WebExtensionPlaceholderMenuItem
 import mozilla.components.browser.state.selector.selectedTab
 import mozilla.components.browser.state.store.BrowserStore
 
 /**
  * Browser menu builder with web extension support. It allows [WebExtensionBrowserMenu] to add
- * web extension browser actions in a nested menu item. If there are no web extensions installed,
- * the web extension menu item would return an add-on manager menu item instead.
+ * web extension browser actions in a nested menu item.
```

Comment:
I would keep the part of the comment that mentions the add-on manager and just add that its visibility is depending on a boolean.